elasticsearch/values.yaml

```
@@ -22,8 +22,9 @@ esMajorVersion: ""
 
 # Allows you to add any config files in /usr/share/elasticsearch/config/
 # such as elasticsearch.yml and log4j2.properties
-esConfig: {}
-#  elasticsearch.yml: |
+esConfig: 
+  elasticsearch.yml: |
+    cluster.deprecation_indexing.enabled: false
```

As this will override the default `elasticsearch.yml` from the Docker image, we need to include `cluster_name` and `network_host`.

```suggestion
  elasticsearch.yml: |
    cluster.name: "docker-cluster"
    network.host: 0.0.0.0
    # required for https://github.com/elastic/helm-charts/issues/1462
    cluster.deprecation_indexing.enabled: false
```